config/database.yml

```diff
@@ -1,6 +1,8 @@
 development:
   adapter: postgresql
-  database: discourse_development
+  database: <%= ENV['DISCOURSE_DB_NAME'] || 'discourse_development' %>
```

Comment:
Well, the idea there is that in development, they wouldn't be in the env, and it would just use what's in the file. While in production, everything is specified in the env.